src/unary.js

```diff
@@ -30,3 +31,25 @@ export const tan = (input) => unary(input, Math.tan);
 export const copy = (input) => unary(input, (x) => x);
 export const reciprocal = (input) => unary(input, (x) => 1 / x);
 export const sqrt = (input) => unary(input, Math.sqrt);
+export const sign = (input) => unary(input, Math.sign);
+
+export const erf = (input) => {
+  /**
+  *reference 1:https://en.wikipedia.org/wiki/Error_function
+  *reference 2:https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-webgl/src/kernels/Erf.ts
+  */
+  const a1 = new Scalar(0.254829592);
+  const a2 = new Scalar(-0.284496736);
+  const a3 = new Scalar(1.421413741);
+  const a4 = new Scalar(-1.453152027);
+  const a5 = new Scalar(1.061405429);
+  const p = new Scalar(0.3275911);
+  const ones = new Scalar(1);
+  input = input.shape.length === 0 ? new Scalar(input.data) : input;
+  const signInput = sign(input);
+  input = abs(input);
+  const t = div(ones, add(ones, mul(p, input)));
+  const y = mul(add(mul(add(mul(add(mul(add(mul(a5, t), a4), t), a3), t), a2), t), a1), t);
+  const result = mul(signInput, sub(ones, mul(y, exp(neg(mul(input, input))))));
+  return result;
```

Comment:
Because `inputFunc` operates on each element, so you may not need to depend on `Scalar` and other binary ops, something like:
```js

const a1 = 0.254829592;
const a2 = -0.284496736;
const a3 = 1.421413741;
const a4 =-1.453152027;
const a5 = 1.061405429;
const p = 0.3275911;

export const erf = (input) => {
    const sign = Math.sign(input);
    const v = Math.abs(input);
    const t = 1.0 / (1.0 + p * v);
    return sign *
        (1.0 -
            (((((a5 * t + a4) * t) + a3) * t + a2) * t + a1) * t *
            Math.exp(-v * v));
}
```

See https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-cpu/src/kernels/Erf.ts

Reply:
Recommend to refer to the tf.js cpu backend implementation which calculates in JavaScript.